src/Framework/DocBlockResolverAwareTrait.php

```diff
@@ -54,7 +54,7 @@ public function __call($method, $parameters = [])
         }
 
         if ($this->hasParentClass()) {
-            /** @psalm-suppress ParentNotFound */
+            /** @var class-string $parentReturn */
             $parentReturn = parent::__call($method, $parameters);
```

Comment:
The parentReturn is not a class-string, but a ?mixed. I am not sure why you did this change 🤔